feat: [CHAOS-925]: Documentation updates for Chaos Module (#504)
harness/developer-hub: docs/chaos-engineering/chaos-faults/kubernetes/node/node-taint.md

---
id: node-taint
title: Node taint
---

> Node taint taints (contaminates) the node by applying the desired effect. 

Reviewer: Why are we talking about contamination here?

- The resources that contain the corresponding tolerations only can bypass the taints.

![Node Taint](./static/images/node-taint.png)

## Usage
<details>
<summary>View the uses of the fault</summary>
<div>

> The fault aims to verify resiliency of applications when the nodes of a container (or deployment) are contaminated.

Reviewer: ```suggestion
The fault aims to verify the resiliency of applications when a certain taint is added to a node.
```
